x-pack/legacy/plugins/epm/server/lib/field.test.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { readFileSync, writeFileSync } from 'fs';
+import path from 'path';
+import { safeLoad } from 'js-yaml';
+import { Field, processFields } from './field';
+
+test('tests loading fields.yml', () => {
+  const fieldsYML = readFileSync(path.join(__dirname, '/tests/fields/base.yml'), 'utf-8');
+
+  const fields: Field[] = safeLoad(fieldsYML);
+
+  processFields(fields);
+
+  // Convert it json for easier comparison of the output
+  const json = JSON.stringify(fields, null, 2);
+  const generatedFile = path.join(__dirname, './tests/fields/base.fields.generate.json');
+
+  // Regenerate the file if `-generate` flag is used
```

Comment:
@jfsiii I moved this back to just using a `-`. With `--` it somehow didn't work. Did not investigate why and just switched it back to `-` which is working for now.

Reply:
It looks like Jest is claiming all options with `--` for itself and doesn't know what to do with 'generate'. I'm not sure if this is by convention or by accident to be honest.

(I'm fine with leaving it as it is right now.)